diff --git a/includes/registration.forms.inc b/includes/registration.forms.inc
index c670d48..f2cf9f3 100644
--- a/includes/registration.forms.inc
+++ b/includes/registration.forms.inc
@@ -974,7 +974,9 @@ function theme_registration_state_overview_form($variables) {
  * Validation helper function to confirm the spaces requested are within limits.
  */
 function _registration_validate_space_request($element, &$form_state, $form) {
-  if (isset($form_state['input']['count']) && $form_state['input']['count'] > $form['#registration_settings']['settings']['maximum_spaces']) {
+  if (isset($form_state['input']['count']) &&
+      $form['#registration_settings']['settings']['maximum_spaces'] &&
+      $form_state['input']['count'] > $form['#registration_settings']['settings']['maximum_spaces']) {
     form_error($element, t('You may not register for more than @count spaces.', array('@count' => $form['#registration_settings']['settings']['maximum_spaces'])));
   };
 }
